AlfrescoContent/AlfrescoContent/Classes/Swaggers/APIs/SearchAPI+SimpleSearch.swift

```diff
@@ -107,7 +107,7 @@ extension SearchAPI {
                                           fields: nil,
                                           sort: sort,
                                           templates: templates,
-                                          defaults: defaults,
+                                          defaults: nil,
```

Comment:
Don't send null, remove it from here

Reply:
nil values don’t go in the request body. It removes automatically if we pass nil.